public/locales/en/bridge.json

```diff
@@ -4,6 +4,7 @@
   "bridgeModal.heading.collecting": "Bridging Collecting",
   "bridgeModal.heading.success": "Bridging Successful",
   "bridgeModal.heading.disclaimer": "Bridging {{typedValue}} {{symbol}}",
+  "bridge.checkoutBridgeUrl": "For more information, visit <0>{{activeBridgeUrl}}↗</0>",
```

Comment:
`activeBridgeUrl` should be changed to `bridgeName`

Reply:
My concern is that we're not showing the bridge name but the active bridge URL, do you still want me to change the naming for this variable? To me, it will be less meaningful with this suggestion.

Reply:
In this case we hint the user that's an URL with the icon and different color. Pasting the URL feels weird, we should use anchor element so we can display text but redirect the user if he clicks. By changing the variable I'm suggesting to change the content also, so we don't show the URL but the bridge name. What would be less meaningful?

Reply:
Maybe we can add more text like `*bridge name* page`. e.g.: Socket bridge page.

Reply:
Ohhhhh I get it now, you want to display the Bridge Name as an anchor redirecting the user into an external link to the Bridge landing. My bad! Will do it!